tmgmt_ckeditor.module

```diff
@@ -57,25 +57,18 @@ function tmgmt_ckeditor_form_tmgmt_job_item_edit_form_alter(&$form, FormStateInt
  * Implements hook_tmgmt_data_item_text_output_alter().
  */
 function tmgmt_ckeditor_tmgmt_data_item_text_output_alter(&$text, $data_item, JobItemInterface $job_item) {
-  if (preg_match_all('/(<([^\btmgmt\b]+[^\s]+)[^>]+?>)([^>]+?[^\/][^>]+>)/', $text, $matches)) {
+  if (preg_match_all('/<((/?[a-z-]+).*?)\s?/?>/', $text, $matches)) {
```

Comment:
my bad, i've forgot to escape "/" in the regex. now is fixed in the commit e7f021e